Firestore/Source/Public/FIRTimestampBehavior.h

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright 2017 Google
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#import <Foundation/Foundation.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+/**
+ * OBC
+ */
+typedef NS_ENUM(NSInteger, FIRTimestampBehavior) {
+  /**
+   * OBC
+   */
+  FIRTimestampBehaviorReturnTimestamp,
+
+  /**
+   * OBC
+   */
+  FIRTimestampBehaviorReturnNativeDate,
+
+  FIRTimestampBehaviorDefault = FIRTimestampBehaviorReturnNativeDate
```

Comment:
I'm not sure we actually want to expose FIRTimestampBehaviorDefault in our public API since its meaning will change in the future...

On that note, maybe we should just drop the enum and have:

settings.enableNewTimestampBehavior = true

And we'll log a warning if they don't set it to true...  and then we'll deprecate it once it's the default, and then delete it eventually...